<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTransactionEditor</class>
 <widget class="QFrame" name="NewTransactionEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>295</height>
   </rect>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,5,0,0">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="horizontalSpacing">
    <number>6</number>
   </property>
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <item row="3" column="0">
    <widget class="QLabel" name="categoryLabel">
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="KMyMoneyDateEdit" name="dateEdit">
       <property name="displayFormat">
        <string notr="true">M/d/yy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hs2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="1" rowspan="2" colspan="2">
    <widget class="QPlainTextEdit" name="memoEdit">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="numberEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Pay to</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QComboBox" name="statusCombo">
     <item>
      <property name="text">
       <string>Cleared</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="payeeEdit">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="autoCompletion">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="KTagContainer" name="tagContainer"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="postdateLabel">
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="memoLabel">
     <property name="text">
      <string>Memo</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="KMyMoneyAccountCombo" name="categoryCombo">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="enterButton">
       <property name="text">
        <string>Enter</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="numberLabel">
     <property name="text">
      <string>Number</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QComboBox" name="costCenterCombo">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="tagsLabel">
     <property name="text">
      <string>Tags</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <layout class="QHBoxLayout" name="hs3">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Amount</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AmountEdit" name="amountEditCredit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Payment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AmountEdit" name="amountEditDebit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Deposit</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>48</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="costCenterLabel">
     <property name="text">
      <string>Costcenter</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="accountLabel">
     <property name="text">
      <string>Account</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="KMyMoneyAccountCombo" name="accountCombo">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KMyMoneyAccountCombo</class>
   <extends>QComboBox</extends>
   <header>kmymoneyaccountcombo.h</header>
  </customwidget>
  <customwidget>
   <class>AmountEdit</class>
   <extends>QLineEdit</extends>
   <header>amountedit.h</header>
  </customwidget>
  <customwidget>
   <class>KMyMoneyDateEdit</class>
   <extends>QDateEdit</extends>
   <header>kmymoneydateedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTagContainer</class>
   <extends>QFrame</extends>
   <header>ktagcontainer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>accountCombo</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>amountEditCredit</tabstop>
  <tabstop>amountEditDebit</tabstop>
  <tabstop>payeeEdit</tabstop>
  <tabstop>numberEdit</tabstop>
  <tabstop>categoryCombo</tabstop>
  <tabstop>costCenterCombo</tabstop>
  <tabstop>statusCombo</tabstop>
  <tabstop>memoEdit</tabstop>
  <tabstop>enterButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
